### PR TITLE
259 copy keys for all

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
Drawing a line under https://trello.com/c/jk3dpUSn/259-add-definition-stage-i-content-placeholder-copy-to-a-copy-system

## Description
- Everyone has at least some copy keys now

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Keys.elm [hunk 1]
```diff
@@ -79,8 +79,63 @@ type Key
...
+    | HelpSelfCategoryLink3
+    | HelpSelfCategoryLink4
+    | HelpSelfCategoryLink5
+    | ToDefinitionFromHelpSelfLink
+    | ToNotAloneFromHelpSelfLink
+    | ToGetHelpFromHelpSelfLink
+    | ToSelfHelpFromSingleCategoryLink
+      -- Help Self Single Category1
+    | HelpSelfCategoryTitle1
+    | HelpSelfCategoryResource1Title1
```
Comment (katjam) [inline src/Copy/Keys.elm]: I think if I put the trailing int after Category, this would be more clear

Comment (Nikomus) [reply src/Copy/Keys.elm]: I think a bit clearer yes.

File: src/Copy/Keys.elm [hunk 2]
```diff
@@ -79,8 +79,63 @@ type Key
...
+    | HelpSelfCategoryResource1QuoteOne1
```
Comment (Nikomus) [inline src/Copy/Keys.elm]: Argh! 😕

**Review by Nikomus — COMMENTED** [review]
I think insetting the number as you suggest will make it clearer, and mean we don't have to mix written and numeric things in the same string

File: src/Copy/Keys.elm (hunk 1 above)
Comment (katjam) [reply src/Copy/Keys.elm]: Good plan. I will move the int.

Comment (katjam) [reply src/Copy/Keys.elm]: Done.

**Review by Nikomus — APPROVED** [review]
This is much clearer! 🌊

**Pull request merged**